great_expectations/core/usage_statistics/execution_environment.py

```diff
@@ -100,11 +101,14 @@ def _get_all_installed_packages(self) -> List[str]:
         """
         if not self._all_installed_packages:
             # Only retrieve once
-            self._all_installed_packages = [
-                item.metadata.get("Name").lower()
-                for item in metadata.distributions()
-                if item.metadata.get("Name") is not None
-            ]
+            package_names: list[str] = []
+            for package in metadata.distributions():
+                package_name = package.metadata.get("Name")
```

Comment:
Do we know the schema of `package.metadata`? This is 3rd party so I guess we're okay here

Reply:
Yep! See here: https://packaging.python.org/en/latest/specifications/core-metadata/#name.